my_gazebo_concert/src/solutions/gazebo.services

```diff
@@ -0,0 +1,13 @@
+# PRIORITIES
+#
+# Turtle pond has scheduler_msgs.Request.DEFAULT_PRIORITY, teleop has 
+# scheduler_msgs.Request.HIGH_PRIORITY. If you want to change the priority
+# override it as below for any of the services:
+#  overrides:
+#    priority: 1000
+
+- resource_name: concert_service_gazebo/gazebo_robot_world
+  overrides:
+    parameters: gazebo_concert/gazebo.parameters
```

Comment:
It still looks like this line should read `parameters: my_gazebo_concert/gazebo.parameters` so that it uses the file in this package instead of the standard package.